frontend/src/plugins/store/index.js

```diff
@@ -11,7 +11,7 @@ Vue.use(Vuex)
 axios.defaults.withCredentials = true
 Vue.use(VueAxios, axios)
 
-export const API_ROOT = process.env.VUE_APP_ROOT_API
+export const API_ROOT = window.environment.API_ROOT_URL
```

Comment:
Eine Alternative wäre, die runtime environment Variablen nur als Fallback zu nutzen:
`export const API_ROOT = window.environment.API_ROOT_URL || process.env.VUE_APP_ROOT_API`

Reply:
So wie du es geschrieben hast ist aber process.env.VUE_APP_ROOT_API der Fallback falls eine Variable im environment.js nicht existiert. Für gewisse Optionen kann ein solches Fallback sinnvoll sein, aber gerade die API root URL ist in jedem Deployment unterschiedlich, und kein sinnvoller Default existiert. Aber z.B. für die BASE_URL im Router habe ich genau so einen Fallback gemacht. Ich würde das also von Variable zu Variable entscheiden.